schema/src/main/jade-tables/analysis_file.table.json

```diff
@@ -6,7 +6,8 @@
     { "name": "content", "datatype": "string", "type": "required" },
     { "name": "file_id", "datatype": "fileref", "type": "required" },
     { "name": "source_file_id", "datatype": "string", "type": "required" },
-    { "name": "source_file_version", "datatype": "timestamp", "type": "required" }
+    { "name": "source_file_version", "datatype": "timestamp", "type": "required" },
```

Comment:
Related change that landed yesterday: We don't need to track `source_file_id` or `source_file_version` now that the info is included in `descriptor`, so we can delete those columns across the board.